components/form/FAutocompleteInput.vue

```diff
@@ -266,38 +274,29 @@ defineExpose<{
   focus,
 });
 
-const formattedFieldValue = computed({
-  get(): string {
-    if (!inputIsActive.value && !!currentOptionMatched.value) {
-      return currentOptionMatched.value.label;
-    }
-    return fieldValue.value;
-  },
-  set(inputValue: string) {
-    fieldValue.value = inputValue;
-    inputIsActive.value = true;
-    currentOptionMatched.value = undefined;
-  },
-});
-
-const inputIsActive = ref(false);
+const inputValue = ref('');
 
 const filterRegex = computed(
-  () => new RegExp(composeSearchRegex(fieldValue.value), 'ig')
+  () => new RegExp(composeSearchRegex(inputValue.value), 'ig')
 );
 
 const matchingOptions = computed(() => {
-  if (!fieldValue.value || !!currentOptionMatched.value) return props.options;
+  if (
+    !inputValue.value ||
+    !!currentOptionMatched.value ||
+    props.preventFiltering
+  )
+    return props.options;
```

Comment:
```suggestion
  if (
    !inputValue.value ||
    !!currentOptionMatched.value ||
    props.preventFiltering
  ) {
    return props.options;
  }
```
when mutli-line if, use braces (we definitely should make prettier do it itself...)